### PR TITLE
Improve jobs screen UX

### PR DESCRIPTION
**Pull request by jamessimo**
## About the PR
This brings minor cosmetic UX improvements to the character creation and job preferences screen. 
Mainly, updated Jobs and Departments ftl lists to be case correct. Updated margins on character name, job name and department name as well as make the job priority buttons uniform width. (see before and after in media) 

This has been tested in all UI scales (100% to 200%) and everything is visible and working as intended.

**Media**
Before: 
oldjobs

After:
newjobs

- [x] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**

:cl:
- tweak: Updated Jobs and Departments to be case-correct 
- tweak: Character job preferences pages UX improvements

### CODE REVIEW

File: Resources/Locale/en-US/job/job-names.ftl [hunk 1]
```diff
@@ -1,86 +1,86 @@
...
+JobResearchDirector = Research Director
+JobSalvageSpecialist = Salvage Specialist
+JobScientist = Scientist
+JobSecurityCadet = Security Cadet
+JobSecurityOfficer = Security Officer
+JobServiceWorker = Service Worker
+JobStationEngineer = Station Engineer
+JobTechnicalAssistant = Technical Assistant
+JobWarden = Warden
+JobBoxer = Boxer
```
Comment (EmoGarbage404) [inline Resources/Locale/en-US/job/job-names.ftl:85]: blease... it's out of order...

Comment (jamessimo) [reply Resources/Locale/en-US/job/job-names.ftl:85]: I'll fix it

**Comment by Visne** [comment]
Hmm, I felt like the lowercasing was pretty consistent with how entities and stuff are also lowercased in the game

**Comment by Elijahrane** [comment]
> Hmm, I felt like the lowercasing was pretty consistent with how entities and stuff are also lowercased in the game

yeah the casing has been consistently like this in all player facing strings

**Comment by Visne** [comment]
Think I will open a PR to revert that part specifically tomorrow

**Pull request merged**